src/main/java/com/cleanroommc/groovyscript/compat/mods/thaumcraft/warp/WarpItemStackExpansion.java

```diff
@@ -0,0 +1,12 @@
+package com.cleanroommc.groovyscript.compat.mods.thaumcraft.warp;
+
+import com.cleanroommc.groovyscript.compat.mods.ModSupport;
+import net.minecraft.item.ItemStack;
+
+public class WarpItemStackExpansion {
+
+    public static void addWarp(ItemStack itemStack, int warp) { ModSupport.THAUMCRAFT.get().warp.addWarp(itemStack, warp); }
+
+    public static void clearWarp(ItemStack itemStack) { ModSupport.THAUMCRAFT.get().warp.removeWarp(itemStack); }
```

Comment:
Wtf is this formatting? Please dont one line functions.
And 2nd you need to register this in the `GroovyScript` class so you can call these methods on item stacks

Reply:
fixed in commit: [d242a72](https://github.com/CleanroomMC/GroovyScript/pull/18/commits/d242a72a6c5de01ec2ae925403af44a157d14bb3)
registered in commit: [9a828eb](https://github.com/CleanroomMC/GroovyScript/pull/18/commits/9a828eb0cc860a95ce6c9d2881874942ef245a82)